src/css/controls/imports/settings-menu.less

```diff
@@ -155,10 +155,15 @@
         }
     }
 
-    .jw-tooltip-settings,
-    .jw-settings-submenu-button .jw-tooltip-captions {
+    .jw-tooltip-settings {
         display: none;
     }
+
+    .jwplayer:not(.jw-flag-small-player) & {
```

Comment:
Currently, the settings menu is after the controlbar in the DOM and the last element in the controls div. If we insert it before the control bar instead of appending it, this bug fixes itself. 
<img width="361" alt="screen shot 2017-10-02 at 5 54 49 pm" src="https://user-images.githubusercontent.com/4603952/31105699-e47961f6-a79a-11e7-9430-08a937d2d53b.png">

Reply:
@egreaves i had considered that option but we would still need css for breakpoints 0 and 1 because the menu covers the entire player on flag-small-player, and the controlbar would be layered on top

Reply:
@karimJWP the controlbar can be set to `display: none` when the settings menu is open at those player sizes.